build.gradle

```diff
@@ -305,8 +315,12 @@ subprojects {
     }
 
     spotbugs {
```

Comment:
I'm curious why we added the `spotbugs` block twice.  We should only need to declare it once in the `subprojects` block.

Reply:
oops. updated!